src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -770,7 +771,7 @@ private void Terminate(bool gracefully)
                 // 9:
                 SafeExecute(() =>
                 {                
-                    var siloStreamProviderManager = (StreamProviderManager)grainRuntime.StreamProviderManager;
+                    var siloStreamProviderManager = (StreamProviderManager) runtimeClient.CurrentStreamProviderManager;
```

Comment:
why switch to runtimeclient?
why not just pass in local siloStreamProviderManager we already have?